go.mod

```diff
@@ -4,10 +4,10 @@ go 1.14
 
 require (
 	github.com/Zilliqa/gozilliqa-sdk v1.2.1-0.20210329093354-1b8e0a7a2e25
-	github.com/btcsuite/btcd v0.20.1-beta
+	github.com/btcsuite/btcd v0.21.0-beta
 	github.com/btcsuite/btcutil v1.0.2
 	github.com/cosmos/cosmos-sdk v0.39.1
-	github.com/ethereum/go-ethereum v1.9.15
+	github.com/ethereum/go-ethereum v1.9.18
```

Comment:
Is this modification necessary? poly depends on go-ethereum heavily, can it be left as is?

Reply:
we will revert the eth version to v1.9.15